lua/lualine.lua

```diff
@@ -210,7 +204,27 @@ local function lualine_set_theme()
   theme_set = M.options.theme
 end
 
-local function statusline(sections, is_focused)
+local function statusline(is_focused)
+  local sections = nil
+  for _, extension in ipairs(M.extensions) do
```

Comment:
You should handle tabline too . It's not handled currently and will break the feature.
Actualy I think it's better to move these logic to `status_dispatch` function . I had left that function for this purpose . Also this function is already doing too much .

Reply:
Yeah, true. I will refactor it today. Thanks for the review :smile:

Reply:
This should be good now